CHANGELOG.md

```diff
@@ -1,4 +1,51 @@
-## 5.28.0 (Unreleased)
+## 5.29.0 (Unreleased)
+
+NOTES:
+* compute: added documentation for md5_authentication_key field in google_compute_router_peer resource ([#17991](https://github.com/hashicorp/terraform-provider-google/pull/17991))
```

Comment:
We may want to note the original release of this field. I think it's valuable to note since it was not documented before, and hard to discover.